src/server/llmQueries.ts

```diff
@@ -7,15 +7,15 @@
  */
 
 const DEBUG_INSTRUCTION = {
-  nb: "Utenfor <answer> taggen så skal du levere refleksjoner på hvorfor du har valgt å inkludere de punktene du har valgt",
-  nn: "Utanfor <answer> taggen så skal du levere refleksjonar på kvifor du har valt å inkludere dei punkta du har valt først.",
+  nb: "Utenfor <answer>-taggen så skal du levere refleksjoner på hvorfor du har valgt å inkludere de punktene du har valgt",
+  nn: "Utanfor <answer>-taggen så skal du levere refleksjonar på kvifor du har valt å inkludere dei punkta du har valt først.",
   en: "Outside of the <answer> tag you are to write reflections as to why you have chosen to include the selected points first.",
 };
 
 const ERROR_INSTRUCTION = {
-  nb: "Om du ikke klarer å levere svaret på formatet <answer> som er spesifisert så skal grunnen skrives i en <ERROR>-tag.",
-  nn: "Om du ikkje klarar å levere svaret på formatet <answer> som er spesifisert skal grunnen skrivast i ein <ERROR>-tag.",
-  en: "If you cannot deliver the answer in the given format <answer>, the reasoning should be written inside a <ERROR>-tag.",
+  nb: "Om du ikke klarer å levere svaret på formatet <answer> som er spesifisert så skal grunnen skrives i en <ERROR> tag.",
+  nn: "Om du ikkje klarar å levere svaret på formatet <answer> som er spesifisert skal grunnen skrivast i ein <ERROR> tag.",
+  en: "If you cannot deliver the answer in the given format <answer>, the reasoning should be written inside a <ERROR> tag.",
```

Comment:
`If you cannot deliver the answer in the provided <answer> format, the reasoning should be written inside an <ERROR> tag.`